examples/ParallaxBarrierEffect.js

```diff
@@ -0,0 +1,115 @@
+// This file is copy pasted from itowns.THREE
```

Comment:
I think you can include it from node_modules/three/... directly in your html. We'll have to check that it'll work in `gh_pages`. Maybe `autodeploy.sh` will need some update.

Reply:
(I mean, I'm sure we don't need the copy-paste)

Reply:
We need the copy-paste, there are some changes below (see  https://github.com/iTowns/itowns/pull/607/files#diff-5715fdf0d63640d34276838bfcde4429R43)

Reply:
I think the far values could be overwritten here: https://github.com/iTowns/itowns/pull/607/files#diff-8f232072bea770740260eca3c875177eR51

Reply:
I don't see how we could access the `_stereo` var.  Plus there is the `dispose` method at the end, that also can't be added like that, as we have no direct access to `_renderTargetL/R`.

Reply:
@peppsac I thought so too (because `render` calls `_stereo.update`), but the far value seem not to be taken fully into account here: https://github.com/mrdoob/three.js/blob/master/src/cameras/StereoCamera.js#L36

(looks like a bug in THREE for me)